src/pages/webhooks/installation.md

```diff
@@ -38,7 +38,7 @@ The following steps apply to both Adobe Commerce on cloud infrastructure and on-
 1. Enable the generated modules:
 
    ```bash
-   bin/magento module:enable AdobeCommerceWebhookPlugins
+   bin/magento module:enable Magento_AdobeCommerceWebhooks Magento_AdobeCommerceWebhooksGenerator
```

Comment:
The modules `Magento_AdobeCommerceWebhooks Magento_AdobeCommerceWebhooksGenerator` should be updated during the step `bin/magento module:enable --all`
But yes, the module name was not correct, should be `Magento_AdobeCommerceWebhookPlugins`